src/main/resources/portal.properties.EXAMPLE

```diff
@@ -220,20 +220,35 @@ ucsc.build=hg19
 oncoprint.defaultview=patient
 
 # OncoPrint driver mutation annotations
-#   oncoprint.custom_driver_annotation.binary.menu_label=Custom driver annotation
-#   oncoprint.custom_driver_annotation.tiers.menu_label=Custom driver tiers
-#   oncoprint.custom_driver_annotation.binary.default=true
-#   oncoprint.custom_driver_annotation.tiers.default=true
-  oncoprint.oncokb.default=true
-  oncoprint.hotspots.default=true
-#  oncoprint.hide_vus.default=true
+# oncoprint.custom_driver_annotation.binary.menu_label=Custom driver annotation
+# oncoprint.custom_driver_annotation.tiers.menu_label=Custom driver tiers
+# oncoprint.custom_driver_annotation.binary.default=true
+# oncoprint.custom_driver_annotation.tiers.default=true
+oncoprint.oncokb.default=true
+oncoprint.hotspots.default=true
+# oncoprint.hide_vus.default=true
 
 # Custom gene sets
 # querypage.setsofgenes.location=file:/<path>
-#
-# Disable cache (for non authorized portals this can sometimes avoid running
-# out of memory with relatively little performance loss)
-# mybatis.cache.enabled=false
+
+# to disable use of Ehcache, set ehcache.cache.enabled to false, use ehcache.xml_configuration=/ehcache-heap-only.xml with ehcache.general_repository_cache.max_bytes_heap=1 with units=B
```

Comment:
This is a bit confusing - Why not say something like:
```
# to disable use of EhCache, uncomment the following four lines and comment all the other lines starting with ehcache 
#ehcache.cache.enabled=false
#ehcache.xml_configuration=/ehchache-heap-only.xml
#ehcache.general_repository_cach.max_bytes_heap=1
#ehcache.general_repository.max_bytes_heap_units=B
```

Reply:
This is no longer necessary. With the latest changes, all a user has to do is to set ehcache.cache_enabled to false --- the rest will then be ignored